src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImplTest.java

```diff
@@ -263,23 +269,55 @@ public void testCreateCollectionExerciseCreatesTheActionPlans() throws Exception
     this.collectionExerciseServiceImpl.createCollectionExercise(toCreate, survey);
 
     // Then
+    verify(actionService, times(1)).createActionPlan("BRES B", "BRES B Case");
+    verify(actionService, times(1)).createActionPlan("BRES BI", "BRES BI Case");
     verify(actionService, times(1)).createActionPlan("BRES B 202103", "BRES B Case 202103");
     verify(actionService, times(1)).createActionPlan("BRES BI 202103", "BRES BI Case 202103");
   }
 
   /**
    * Tests that creating a collection exercise for which action plans exists does not try to create action plans
-   * @throws Exception
+   * @throws Exception general exception
    */
   @Test
-  public void testCreateCollectionExerciseExistingActionPlans() throws Exception {
+  public void testCreateCollectionExerciseExistingDefaultActionPlans() throws Exception {
     // Given
     CollectionExerciseDTO toCreate = FixtureHelper.loadClassFixtures(CollectionExerciseDTO[].class).get(0);
     CollectionExercise collectionExercise = FixtureHelper.loadClassFixtures(CollectionExercise[].class).get(0);
     collectionExercise.setExerciseRef(toCreate.getExerciseRef());
     when(collexRepo.saveAndFlush(any())).thenReturn(collectionExercise);
     SurveyDTO survey = FixtureHelper.loadClassFixtures(SurveyDTO[].class).get(0);
     when(this.surveyService.findSurvey(UUID.fromString(toCreate.getSurveyId()))).thenReturn(survey);
+    ActionPlanDTO actionPlanDTO = new ActionPlanDTO();
```

Comment:
:art: Would consider putting a couple of blank lines in this and the next test (probably after the when statements) to make it more readable, but maybe that's just me reading it in GitHub rather than an IDE.

Reply:
:+1: Will add a couple more comments to to make it clearer what we're mocking